File: Beginner/From heaven to earth (ELEVSTRS)/heaven.py
t = int(input())

while t:
    N, S, E = map(int, input().split())

    x = (N*(2**(1/2)))/S
    y = 2*N/E

    if y<x:
        print("Elevator")
    else:
        print("Stairs")

    t = t-1

File: Beginner/From heaven to earth (ELEVSTRS)/heaven.cpp
#include <iostream>
#include<math.h>
using namespace std;

int main() {

	int t;
	cin>>t;
	while(t--) {
	    float N, S, E;
	    float x, y;
	    cin >> N >> S >> E;
	    x = (sqrt(2)*N)/S;
	    y = 2*N/E;

	    if(x>y)
	        cout<<"Elevator"<<endl;
	    else
	        cout<<"Stairs"<<endl;
	}
	return 0;
}
